firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/analytics/AnalyticsConnectorReceiver.java

```diff
@@ -66,6 +67,29 @@ public boolean register() {
     analyticsConnectorHandle =
         analyticsConnector.registerAnalyticsConnectorListener(CRASHLYTICS_ORIGIN, this);
 
+    if (analyticsConnectorHandle == null) {
+      Logger.getLogger()
+          .d(
+              "Could not register AnalyticsConnectorListener with origin "
+                  + "\""
+                  + CRASHLYTICS_ORIGIN
```

Comment:
I don't think we want to expose this implementation detail to the user. Maybe something more generic like "Could not register analytics connector with Crashlytics origin"?

Reply:
Sure, NBD. In general I'm not opposed to exposing inconsequential (to the customer) implementation details in debug logs, but it doesn't really matter what the message is here, so long as we know how to interpret it. :)